src/_guides/language/effective-dart/style.md

```diff
@@ -444,9 +463,8 @@ if (isWeekDay) {
 }
 {% endprettify %}
 
-There is one exception to this: an `if` statement with no `else` clause where
-the entire `if` statement and the then body all fit in one line. In that case,
-you may leave off the braces if you prefer:
+**Exception:** An `if` statement with no `else` clause that fits entirely on one
```

Comment:
All the other exceptions start with sentences, so let's do that here, too:

When you have an `if` statement with no `else` clause and the whole `if` statement fits on one line,
you can omit the braces if you prefer: